src/auro-radio.js

```diff
@@ -57,6 +57,13 @@ class AuroRadio extends LitElement {
     };
   }
 
+  handleChange(event) {
```

Comment:
Can you update the "should fire a input event with correct data" test to also check for the change event?